README.md

```diff
@@ -0,0 +1,13 @@
+# dockerfiles
+
+This repository hold all the dockerfiles useful @ Objectif Libre.
+
+## CloudKitty
```

Comment:
The dockerfiles aren't included with this commit. Remove this section and add it at the same time as the Dockerfiles.